apps/test/unit/javalab/PlaygroundTest.js

```diff
@@ -426,6 +431,11 @@ describe('Playground', () => {
     expect(onJavabuilderMessage).to.not.have.been.called;
   });
 
+  it('sets isProgramRunning to false on stop', () => {
```

Comment:
nit: verify onOutputMessage and onNewlineMessage mocks here too?

Reply:
done!